src/main/java/aztech/modern_industrialization/items/SteamDrillItem.java

```diff
@@ -290,7 +291,7 @@ public boolean mineBlock(ItemStack stack, Level world, BlockState state, BlockPo
             }
         });
         totalDrops.forEach(itemStack -> {
-            Block.popResource(world, miner.blockPosition(), itemStack);
+            ItemHandlerHelper.giveItemToPlayer(p, itemStack);
```

Comment:
This has incompatibility with the Bank Storage mod, where it doesn't pick up the items mined by the drill in 3x3 mode. I'm talking with the dev to see if we can either get a workaround or a fix for this. Just something to keep in mind :)

Reply:
Worked it out with them and they said they'll publish a fix in <1 week